packages/bot-web-ui/src/components/quick-strategy/config.ts

```diff
@@ -1,6 +1,14 @@
 import { config as qs_config } from '@deriv/bot-skeleton';
 import { localize } from '@deriv/translations';
-import { D_ALEMBERT, MARTINGALE, OSCARS_GRIND, REVERSE_D_ALEMBERT, REVERSE_MARTINGALE, _1_3_2_6 } from './descriptions';
+// import { _1_3_2_6 } from './descriptions/1_3_2_6';
```

Comment:
can we remove this?

Reply:
Sure @prince-deriv. This PR is merged already, Will remove in the open PR.